.github/workflows/rust.yml

```diff
@@ -19,4 +19,4 @@ jobs:
     - name: Build
       run: cargo build --verbose
     - name: Run tests
-      run: cargo test --verbose
+      run: cargo test --all-features --verbose
```

Comment:
This should probably still test with and without `--all-features`, just to make sure no regressions occur with the default feature flags too.

Reply:
```suggestion
      run: cargo test --verbose
      run: cargo test --all-features --verbose
```